README.md

```diff
@@ -1,5 +1,145 @@
 
-# Specification 'origin-policy'
+# Explainer: Origin Manifest
 
-This is the repository for origin-policy. You're welcome to contribute! Let's make the Web rock our socks
-off!
+**tl;dr** Origin Manifest (formerly Origin Policy) is a web platform mechanism
+that aims to shift configuring the origin from web applications to the origin
+itself. For this the origin provides a manifest file in a predefined well-known
+location which browsers can load and apply.
+
+**Note:** Though the current draft still uses the name Origin Policy, we use the
+new term Origin Manifest here to avoid confusion with the Same Origin Policy.
+
+
+## The Problems and Goals
+
+### Configurations with origin-wide effects
+**Problem:** Some configurations, e.g. like HTTP headers, affect an entire
+  origin even though they might have been set through only a sub-resource load.
+  Misconfiguration of font resource header can have dramatic effects for the
+  origin, e.g. a wrong HPKP value.
+
+**Goals:** Configurations for web applications with effects on entire origin in
+  a more structured way to minimize the chances for origin misconfiguration
+  through individual web applications sharing the same origin.
+
+### HTTP header redundancy
+**Problem 1:** Same HTTP headers have to be sent again and again without the
+  values actually to change. Sometimes these headers add significant number of
+  bytes, e.g. CSP.
+
+**Problem 2:** Some HTTP headers are sent with every response even though their
+  effect has a lifetime, e.g. HSTS. Within the valid time frame these headers
+  do not need to be repeated.
+
+**Goal:** Mechanism to download configurations like HTTP header values only
+  once. These values are only overridden when needed.
```

Comment:
If you don't below, linking to @mnot's https://mnot.github.io/I-D/site-wide-headers/ would be valuable, as it probably does a better job of addressing this specific problem in isolation.

Reply:
Since this is an explainer, I'd rather not go into long comparisons here. Or should I? I will add the classic "Related Work" section to honor his draft.

Reply:
Feel free to steal text from it, it's unlikely to be published.